mods/tuxemon/maps/sphalian_town_house.tmx

```diff
@@ -54,7 +54,7 @@
   </object>
   <object id="24" name="Watch TV" type="event" x="32" y="96" width="16" height="16">
    <properties>
-    <property name="act1" value="dialog You watch some TV with your beloved Lv ${{monster_0_level}} ${{monster_0_name}}."/>
+    <property name="act1" value="dialog You watch some TV with your beloved Lv ${{monster_0_level}} ${{monster_0_name}}.,0"/>
```

Comment:
Last time i tried this without any monsters the game crashed.  has that been fixed?

Reply:
Just retested with the latest commit: No crash either when there's no parameter nor when an invalid one is used.